pkg/kubelet/server.go

```diff
@@ -148,6 +149,8 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.mux.HandleFunc("/logs/", s.handleLogs)
 	s.mux.HandleFunc("/containerLogs/", s.handleContainerLogs)
 	s.mux.Handle("/metrics", prometheus.Handler())
+	// The /runningpods endpoint is used for testing only.
+	s.mux.HandleFunc("/runningpods", s.handleRunningPods)
```

Comment:
I think that one could argue this one either way.  I guess that we should at least be consistent one way or the other, which we're not currently.  See "syncloop" vs "containerLogs" and "portForward".